src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -782,6 +824,10 @@ void BLEManagerImpl::HandleConnect(struct bt_conn * conId, uint8_t err)
 {
     ChipDeviceEvent event;
 
+    // Don't handle BLE connecting events when it is not related to CHIPoBLE
+    if (!ConnectivityMgr().IsBLEAdvertisingEnabled())
```

Comment:
I think we should handle that condition in the `kPlatformZephyrBleConnected` event handler. This function can be called in the context of some Zephyr's thread while state of the BLE Manager can be changed in the CHIP stack thread.

Reply:
As discussed F2F it can't be moved and for now there is no risk to be hurt by context changes.